frontend/src/__tests__/AdminDomains.test.js

```diff
@@ -254,7 +254,7 @@ describe('<AdminDomains />', () => {
       )
 
       const domainUrlInput = await findByRole('textbox', {
-        name: 'Domain URL',
+        label: 'Search by Domain URL',
```

Comment:
This should still use the `name` option. `label` isn't an option, so findByRole just matches to the textbox anyway since it's the only textbox there